docs/roadmap.md

```diff
@@ -6,39 +6,40 @@ For a general overview, see the [readme](https://github.com/microsoft/ProjectReu
 Windowing | ❌ Not Supported | ❌ Not Supported | ✅ Supported |
 [Push Notifications](https://github.com/microsoft/ProjectReunion/blob/main/specs/PushNotifications/PushNotifications-spec.md) | ❌ Not Supported | ❌ Not Supported | 🔄 Experimental |
-Deploy Options: |  
-• In-app | ✅ Supported | ✅ Supported | ✅ Supported | 
-• Fx Package | ✅ Supported | ✅ Supported | ✅ Supported | 
+Deploy Options: |
+• In-app | ✅ Supported | ✅ Supported | ✅ Supported |
+• Fx Package | ✅ Supported | ✅ Supported | ✅ Supported |
 
 ### Unpackaged Desktop Apps
 
 As an unpackaged desktop app (WPF, WinForms, Win32, or console, not using [MSIX](https://docs.microsoft.com/windows/msix/overview)), here's what and when you (tentatively) can expect from the Windows App SDK!
 
- 🌟 |       Windows App SDK 0.5      |         Windows App SDK 0.8   |       Windows App SDK 1.0 (Q4 2021)    | 
+ 🌟 |       Windows App SDK 0.5      |         Windows App SDK 0.8   |       Windows App SDK 1.0 (Q4 2021)    |
 -- | ---------------------------------- | ------------------------------ | ------------------------------- |
-[WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3) | ❌ Not Supported | ❌ Not Supported | ✅ Supported | 
-[Text rendering](https://docs.microsoft.com/en-us/windows/apps/project-reunion/dwritecore) | ❌ Not Supported | 🔄 Experimental | ✅ Supported | 
-[Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview) | ❌ Not Supported| 🔄 Experimental | ✅ Supported | 
-[App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111) | ❌ Not Supported | 🔄 Experimental | ✅ Supported | 
-[Power state notifications](https://github.com/microsoft/ProjectReunion/issues/111) | ❌ Not Supported | 🔄 Experimental | ✅ Supported | 
-Windowing | ❌ Not Supported | ❌ Not Supported | ✅ Supported | 
+[WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3) | ❌ Not Supported | ❌ Not Supported | ✅ Supported |
+[Text rendering](https://docs.microsoft.com/en-us/windows/apps/project-reunion/dwritecore) | ❌ Not Supported | 🔄 Experimental | ✅ Supported |
+[Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview) | ❌ Not Supported| 🔄 Experimental | ✅ Supported |
+[Dynamic Dependencies](https://github.com/microsoft/WindowsAppSDK/blob/main/specs/dynamicdependencies/DynamicDependencies.md) | ❌ Not Supported | 🔄 Experimental | ✅ Supported |
+[App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111) | ❌ Not Supported | 🔄 Experimental | ✅ Supported |
+[Power state notifications](https://github.com/microsoft/ProjectReunion/issues/111) | ❌ Not Supported | 🔄 Experimental | ✅ Supported |
+Windowing | ❌ Not Supported | ❌ Not Supported | ✅ Supported |
 [Push Notifications](https://github.com/microsoft/ProjectReunion/blob/main/specs/PushNotifications/PushNotifications-spec.md) | ❌ Not Supported | ❌ Not Supported | 🔄 Experimental |
```

Comment:
Not your text, but with this can we update Push Notifications to use sentence casing?

```suggestion
[Push notifications](https://github.com/microsoft/ProjectReunion/blob/main/specs/PushNotifications/PushNotifications-spec.md) | ❌ Not Supported | ❌ Not Supported | 🔄 Experimental |
```